src/screens/library/LibraryScreen.tsx

```diff
@@ -139,6 +140,18 @@ const LibraryScreen = ({ navigation }: LibraryScreenProps) => {
     setFalse: closeSetCategoryModal,
   } = useBoolean();
 
+  function openRandom() {
+    const novels = library[index].novels;
+    const randomNovel = novels[Math.floor(Math.random() * novels.length)];
```

Comment:
Doesn't this apply for current category only?

Reply:
yes, it also respects current filter applied

for reference, my workflow with it is: open want to read category -> open random -> move to reading category